src/main/java/dev/snowdrop/jira/atlassian/Args.java

```diff
@@ -35,4 +35,7 @@
 
     @Parameter(names = "-git", description = "GitHub reference")
     public String gitRef;
+
+    @Parameter(names = "-watcher_list", description = "JIRA Watcher List CSV")
```

Comment:
Please add a default value so that we only need to specify this option when wanting to override the default (which should be the whole team).